### ISSUE TITLE
Feature request: Blinking cursor for EditView.

### ISSUE THREAD

**Issue by jaygazula27** [gyscos/cursive#486]
This is a feature request.

# Problem description
Currently, whenever a EditView takes focus, the cursor is rendered but does not blink. Perhaps this is not possible as it might be too expensive. But it would be nice to have since it's easier to find the view with the focus especially when there is a lot going on the screen.

# Environment

* Operating system: Ubuntu 20.04.1
* Backend used: Crossterm
* Current locale: en_US.UTF-8
* Cursive version: 0.15.0

**Comment by gyscos** [comment]
Hi, and thanks for the report!

Sounds like we should first add "blinking" as an available effect, and then improve the theming support for the cursor.

**Comment by gitatomic** [comment]
I second that. Sounds like a great feature however small.

**Comment by jomag** [comment]
Yes, I did not notice this before, but it seems like cursive does not use the native cursor? I know that other TUI libs, such as urwid, supports the native cursor.
